arangod/RestHandler/RestAdminStatisticsHandler.cpp

```diff
@@ -56,6 +56,9 @@ RestStatus RestAdminStatisticsHandler::execute() {
   }
 
   if (_request->requestPath() == "/_admin/statistics") {
+    if (_request->value("dont-count-this-request") == "true") {
+      _statistics.SET_IGNORE();
+    }
     getStatistics();
```

Comment:
Make this maintenance Mode only ?

Reply:
This is required by the integration tests. Is the maintenance mode enabled in your CI build for these tests? (I assume you mean `ARANGODB_ENABLE_MAINTAINER_MODE`)